src/main/resources/view/ResultDisplay.fxml

```diff
@@ -5,6 +5,6 @@
 <?import javafx.scene.layout.StackPane?>
 <StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1">
   <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" />
-    <Label fx:id="directory" styleClass="result-display" StackPane.alignment="BOTTOM_RIGHT">
-    </Label>
+  <Label fx:id="sortType" styleClass="status-msg" StackPane.alignment="BOTTOM_LEFT"/>
```

Comment:
occasionally, the sort type will be overlapped with other result display message. Might need to be improved

Reply:
Will fix in next PR